crates/hir/src/semantics.rs

```diff
@@ -324,6 +328,53 @@ impl<'db> SemanticsImpl<'db> {
         Some(node)
     }
 
+    pub fn expand_allowed_builtins(&self, macro_call: &ast::MacroCall) -> Option<SyntaxNode> {
+        let sa = self.analyze_no_infer(macro_call.syntax())?;
+
+        let macro_call = InFile::new(sa.file_id, macro_call);
+        let file_id = if let Some(call) =
+            <ast::MacroCall as crate::semantics::ToDef>::to_def(self, macro_call)
+        {
+            call.as_macro_file()
+        } else {
+            sa.expand(self.db, macro_call)?
+        };
+        let macro_call = self.db.lookup_intern_macro_call(file_id.macro_call_id);
+
+        match macro_call.def.kind {
+            hir_expand::MacroDefKind::BuiltIn(
+                _,
+                BuiltinFnLikeExpander::Cfg
+                | BuiltinFnLikeExpander::StdPanic
+                | BuiltinFnLikeExpander::Stringify
+                | BuiltinFnLikeExpander::CorePanic
+                | BuiltinFnLikeExpander::FormatArgs,
```

Comment:
We can skip expanding this one as well now, it expands to unstable syntax so the user shouldn't need to see it

Reply:
hey @Veykril Are you referring to all 4 or some subset specifically ?

Reply:
I assume you meant `format_args`. I have removed it.